node/packages/aws-lambda-sdk/instrument/index.js

```diff
@@ -138,27 +136,16 @@ const closeTrace = async (outcome, outcomeResult) => {
   };
 
   try {
+    const endTime = process.hrtime.bigint();
```

Comment:
Just curious why the `responseStartTime` and `endTime` became one? 🤔

Reply:
Good question, I should have probably split this into another commit.

`responseStartTime` was used to measure the overhead of the extension, and was measured around the same moment as `endTime`, so this is a simplification to rely on single timestamp (below now we write summary by subtracting `endTime` instead of `responseStartTime`)